src/lib/is-array.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = Array.isArray || function (arg) {
+  return Object.prototype.toString.call(arg) === '[object Array]';
```

Comment:
We should put the mdn link as a comment. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#Polyfill